src/ducks/Alert/hooks/useLastPrice.js

```diff
@@ -0,0 +1,25 @@
+import { useMemo } from 'react'
+import { useQuery } from '@apollo/react-hooks'
+import { LAST_PRICE_QUERY } from './queries'
+
+export const useLastPrice = slugTitle => {
+  const { data, loading } = useQuery(LAST_PRICE_QUERY, {
+    variables: {
+      slug: slugTitle
+    },
+    skip: !slugTitle
+  })
+
+  const value = useMemo(() => {
+    if (data && data.getMetric.timeseriesData[0]) {
+      return data.getMetric.timeseriesData[0].value
+    }
+
+    return 0
+  }, [data])
+
+  return {
+    data: value,
+    loading
+  }
```

Comment:
Use the [`aggregatedTimeseriesData`](https://github.com/santiment/app/blob/8b14341b0d0532250f193bff6e506080c328f736/src/ducks/Studio/AdvancedView/PriceHistogram/gql.js#L20) with the `aggregation: LAST` parameter or [`Project.priceUsd`](https://api.santiment.net/graphiql?query=%7B%0A%20%20projectBySlug(slug%3A%22bitcoin%22)%20%7B%0A%20%20%20%20priceUsd%0A%20%20%7D%0A%7D)